### ISSUE TITLE
Add .coafile configuration via plugin

### ISSUE THREAD

**Issue by sims1253** [coala/coala-atom#18]
It would be nice to be able to add/remove bears and set options for the bears via the atom plugin GUI.
Something like this for every Bear:
![mockup](https://cloud.githubusercontent.com/assets/6530123/13036826/9509b43e-d372-11e5-8c13-5450861678b1.png)

As a first try the system wide .coafile could be used as the atom plugin itself is optional. If possible though it would be nice to be able to make these different for every project.

**Comment by sils** [comment]
Yeah I think we could just edit the appropriate config there (found by find-config as that's the one we use)
